configs/components/puppet-ca-bundle.json

```diff
@@ -1,4 +1,4 @@
 {
-  "url": "git://github.com/puppetlabs/puppet-ca-bundle.git",
+  "url": "git@github.com:puppetlabs/puppet-ca-bundle.git",
```

Comment:
It is public. I'm not sure about ssh keys. The URLs are inconsistent; currently pxp-agent and cpp-pcp-client are the only others using `git@`.